TerriaJS/terriajs
# Add idealZoom trait to mappable.

### What this PR does

Fixes #6119 
  
### Test me
- Go to dev site. 
- Add `3D Imagery` -> `Anglesea 3D Photomesh`.
- Click on `IDEAL ZOOM`. You should see a river and bridge.

### Checklist
